src/main/java/it/pagopa/pn/paperchannel/middleware/queue/consumer/handler/RECAG012MessageHandler.java

```diff
@@ -7,47 +7,48 @@
 import lombok.extern.slf4j.Slf4j;
 import reactor.core.publisher.Mono;
 
-//effettuare PUT di una nuova riga in accordo con le specifiche
-//in caso di RECAG012 la document date coincide con la data di produzione dell’evento
-//In caso di ricezione di RECAG012 il documentType non sarà presente, il campo DOCUMENT_TYPE  va quindi popolato con una stringa fissa
-@Slf4j
-public class RECAG012MessageHandler implements MessageHandler {
-
-    private static final String PARTITION_KEY_PREFIX = "DEMAT";
-    private static final String SORT_KEY_PREFIX = "RECAG012";
+import java.util.Optional;
 
-    private static final String RECAG012_DELIMITER = "##";
+import static it.pagopa.pn.paperchannel.utils.MetaDematUtils.*;
 
-    private final EventMetaDAO eventMetaDAO;
+// effettuare PUT di una nuova riga in accordo con le specifiche
+// Il RECAG012 è considerato come entità logica un metadata
```

Comment:
scriverei un pò più nel dettaglio cosa deve fare questa classe, "in accordo con le specifiche" non aiuta per niente a capire che vuol dire :)